### DISCUSSION TITLE
Running both BirdnetPi and Birdnet Analyzer on the same raspberry

### DISCUSSION THREAD

**Discussion by nopedose** [mcguirepr89/BirdNET-Pi#996] [General]
Hello,
 I would like to use 2 programs simultaneuously.

currently, i am running BirdnetPi on RaspbianOS and when i go to the localhost in my browser, I can access it.

I would like to add BirdnetPi Analyzer on the same RaspberryPi, what should i do to not replace the localhost with the new program so i can access both programs in my browser? (using ports probably, but i don't know how)

I would also actually like to install MotionAI on the same raspberry.

Unfortunately I'm a linux noob.

Thanks in advance!

**Comment by srd424** [comment]
Probably not the easiest thing to achieve at the moment, unfortunately, particularly if you're new to Linux. If ever get around to porting my docker stuff back to arm64 that would help on the BirdNET-Pi side, at least, but someone would need to dockerize BirdNet Analyzer.

Having said that, have you looked to see if https://github.com/mmcc-xx/BirdCAGE would be a workable replacement for one or both of those projects?

If you're up for a bit of a learning curve LXC/LXD might be the most efficient way to stack up multiple projects on a Pi - should be lighter weight than virtual machines.

**Comment by nopedose** [comment]
Thank you for your answer.
BirdCAGE looks great, thanks!
I looked into how to install it. It runs on docker, which runs on home
assistant, which runs on linux.
I dont know any of those things. (except linux, but not much experience) so
indeed a very steep learning curve. Very overwhelming and i'm unsure about
how and where to start.

But you talk about lxc/lxd, whould this be easier than docker and would i
still be mostly able to follow the installation instructions if i go this
route?




Op za 12 aug. 2023 17:19 schreef srd424 ***@***.***>:

**Reply by srd424** [reply]
I'm not sure if BirdCAGE actually *needs* Home Assistant, I've not looked at it in too much detail.

Building images with docker can be complicated, but running them isn't necessarily too bad.

LXD might not have too steep a learning a curve - I think it basically creates containers that the equivalent of a lightweight VM, so you should be able to run the installer without too much difficulty.

**Comment by nopedose** [comment]
Okay, thank you!



Op zo 13 aug. 2023 10:38 schreef srd424 ***@***.***>:
